### PR TITLE
Add `Start Metals` button

### PR DESCRIPTION
**Pull request by kpodsiad**

### CODE REVIEW

**Review by tgodzik — APPROVED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -125,8 +126,8 @@
     "viewsWelcome": [
       {
         "view": "metalsPackages",
-        "contents": "No Scala project found.\n[New Scala project](command:metals.new-scala-project)",
-        "when": "workbenchState != empty"
+        "contents": "No Scala project found. Create a new one or start Metals manually.\n[New Scala project](command:metals.new-scala-project)\n[Start Metals server](command:metals.start-server)",
```
Comment (tgodzik) [inline package.json:129]:
```suggestion
        "contents": "No Scala project found. Create a new one or start Metals manually.\n[New Scala project](command:metals.new-scala-project)\n[Start Metals](command:metals.start-server)",
```

**Review by tgodzik — CHANGES_REQUESTED** [review]
Actually, maybe let's make it simpler?

**Review by tgodzik — APPROVED** [review]
Thanks!

**Review by Arthurm1 — APPROVED** [review]
Works a treat - thanks @kpodsiad

**Pull request merged**